src/pages/AddLiquidityV2/index.tsx

```diff
@@ -660,12 +707,21 @@ export default function AddLiquidity() {
           tabsCount={positionsState.length}
           selectedTab={pIndex}
           onChangedTab={index => setPositionIndex(index)}
-          onAddTab={onAddPosition}
-          onRemoveTab={onRemovePosition}
+          onAddTab={onAddPositionEvent}
+          onRemoveTab={onRemovePositionEvent}
           showChart={showChart}
-          onToggleChart={(newShowChart: boolean | undefined) =>
-            setShowChart(showChart => (typeof newShowChart !== 'undefined' ? newShowChart : !showChart))
-          }
+          onToggleChart={(newShowChart: boolean | undefined) => {
+            const newValue = typeof newShowChart !== 'undefined' ? newShowChart : !showChart
+            if (newValue && tokenA?.symbol && tokenB?.symbol && account) {
+              mixpanelHandler(MIXPANEL_TYPE.ELASTIC_ADD_LIQUIDITY_CLICK_PRICE_CHART, {
+                token_1: tokenA?.symbol,
+                token_2: tokenB?.symbol,
+                networkName: networkInfo.name,
+                wallet_address: account,
```

Comment:
remove 2 params from handler, it should be default in useMixpanel hook

Reply:
done